tools/check-tracking-number-and-date.mjs

```diff
@@ -0,0 +1,48 @@
+import { parse as parseTOML } from 'toml';
+import { readFileSync } from 'fs';
+import { campaignInfoToCampaignConfig } from '../webpack/convert_info_to_type.js';
+
+const campaignConfig = parseTOML( readFileSync( 'campaign_info.toml', 'utf-8' ) );
+const configObject = campaignInfoToCampaignConfig( campaignConfig );
+
+for ( const configEntry of Object.entries( configObject ) ) {
+	const configValues = configEntry[ 1 ];
```

Comment:
```suggestion
for ( const configValues of Object.values( configObject ) ) {
```

If you're omitting the channel names anyway, you can make this simpler. Naming-wise I'd also suggest using `campaignConfig` instead of the rather generic `configValues` and renaming `campaignConfig` in line 5 to `rawConfigForAllCampaigns` or something similar.